datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_traps_db.py

```diff
@@ -337,4 +342,104 @@ def get_oid_and_descr(var_name, mib_name, search_locations=None):
 
     if var_name not in file_content:
         raise VariableNotDefinedException()
-    return file_content[var_name]['oid'], file_content[var_name].get('description', '')
+
+    # grab enum if it exists in-line
+    enum = file_content[var_name].get('syntax', {}).get('constraints', {}).get('enumeration', {})
+    # if there is no enum in-line, check for type definition and enum in the same MIB and its imports
+    if not enum:
+        var_type = file_content[var_name].get('syntax', {}).get('type', '')
+        if var_type:
+            try:
+                enum = get_enum(var_type, mib_name, search_locations)
```

Comment:
Seems like we simplify this with:
 right? Any reason not to do it ?
```suggestion
    enum = get_enum(var_name, mib_name, search_locations)
```

Reply:
Unfortunately it's not quite that simple. In the case of an in-line enum, the JSON path is slightly different than if it's on a textual convention.

For in-line: mib[var_name][syntax][constraints][enumeration]
For textual conventions: mib[var_name][type][constraints][enumeration]